### PR TITLE
node-density-cni update

### PR DESCRIPTION
**Pull request by jtaleric**
Have node-density-cni match similar to node-density-heavy.

### CODE REVIEW

File: ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-ref.yaml [hunk 1]
```diff
@@ -15,9 +15,17 @@ ref:
     documentation: |-
       File in which we will parse to find uuid, start and end time
   - name: PODS_PER_NODE
-    default: "200"
+    default: "100"
```
Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-ref.yaml:18]: do we need to lower this?

Comment (jtaleric) [reply ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-ref.yaml:18]: @paigerube14 this was to mimic what we are doing with node-density heavy --- Last I heard we wanted those things to mimic each other. https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-ref.yaml#L21

**Pull request merged**